### PR TITLE
Added screenshots to local file directory

### PR DESCRIPTION
**Pull request by Auxy6858**
Fixed images not appearing

### CODE REVIEW

**Comment by Auxy6858** [comment]
@jonbarrow I added the images locally

**Comment by Auxy6858** [comment]
This is a new PR, I closed the other one

File: blogposts/9-29-21.md [hunk 1]
```diff
@@ -187,37 +187,37 @@ In the meantime, check out our [Patreon](https://patreon.com/pretendonetwork) to
...
-![](https://cdn.discordapp.com/attachments/868275684630470697/892943312380645386/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/Doodle.png)
 > Drawing Post with Reply
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892943575560638534/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/UserMenu.png)
 > User Menu Screen
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892943834521161788/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/Notifications3DS.png)
```
Comment (jonbarrow) [inline blogposts/9-29-21.md:222]: None of these links will work. The `public` folder is not exposed to the client, only the files/folders *within* the `public` folder

Also we do not use relative links here. You can see the other blog posts for how to do this

**Review by jonbarrow — CHANGES_REQUESTED** [review]
Please test your changes locally before making PRs, the current changes would not work when deployed

**Comment by jonbarrow** [comment]
Also your changes are on the wrong branch. Do not PR into the master branch, we have a dev branch for changes

**Pull request closed without merging**